django/webapp/models.py

```diff
@@ -37,7 +37,6 @@ class CannotGetAliya(Exception):
 
 class Synagogue(models.Model):
     name = models.TextField()
-    admins = models.ForeignKey(Group, on_delete=models.CASCADE)
     member_creator = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
do we still need this mechanism?

Reply:
why not?
it's a very advanced feature and it was silly to write it that early, but we will need it if we will want to let someone add member without being able to do nothing else

Reply:
just because its sort of half implemented, but we can leave it if you want